dev/io.openliberty.microprofile.telemetry.1.1.internal/bnd.bnd

```diff
@@ -54,11 +54,13 @@ Import-Package: \
   *
 
 Export-Package: \
-  io.openliberty.microprofile.telemetry.internal.rest;version=1.1;thread-context=true
-
+	io.openliberty.microprofile.telemetry.internal.cdi;version='1.0',\
+	io.openliberty.microprofile.telemetry.resources;version='1.0',\
+	io.openliberty.microprofile.telemetry.rest;version='1.0',\
```

Comment:
I can't immediately find anything in a package called `io.openliberty.microprofile.telemetry.rest`. I can see the internal version but not this (which I would interpret as an API package)